test/simple-memory-corruption/uninitialized_read_large.c

```diff
@@ -0,0 +1,14 @@
+#include <stdlib.h>
+
+#include "../test_util.h"
+
+OPTNONE int main(void) {
+	char *p = malloc(128 * 1024);
+	for (unsigned i=0; i<8; i++) {
```

Comment:
Please add spaces around the `=` and `<` operators for consistency.

Reply:
Should also use 4 space indents like we use elsewhere.

Reply:
> Please add spaces around the `=` and `<` operators for consistency.

Also in `uninitialized_read_small.c`

Reply:
It would be nice to add a `format` target in the Makefile making use of https://clang.llvm.org/docs/ClangFormat.html